public/schemas/searchFilters/announcements-search.json

```diff
@@ -0,0 +1,78 @@
+{
+  "type": "filterCreationSchema",
+  "listingType": "announcements",
```

Comment:
The content of this file looks to me like filters for the housing listing type. But the filename and listingType refer to announcements ?

Reply:
Yeah, it doesn't make sense currently.  I briefly mentioned this in the Pull Request's description. All the filter types are in the `announcements-search.json` just for testing purposes. Before I merge this PR I will clear the `announcements-search.json` file of the filters. UI and logic of the filters should be working, but since they are not connected to GraphQL yet they really shouldn't be displayed at all.